tools/AutoTuner/installer.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Get the directory where the script is located
+script_dir="$(dirname "${BASH_SOURCE[0]}")"
+cd $script_dir
+
+# Check if the Python version is 3.8
+required_version="3.8"
+if command -v python${required_version} &> /dev/null ; then
+    echo "Found python${required_version}"
+else
+    echo "Error: Python ${required_version} is required."
```

Comment:
What OS ships *specifically* with Python 3.8?

Reply:
Doesn't work on Ubuntu 23.10

Reply:
Shouldn't this be 3.8 or newer?

Reply:
We settled on 3.8 because Ubuntu22 is prepacked with 3.10, Ubuntu20 with 3.8, and Centos7 with 3.6, so it happens to be the middle ground. Going forward in future PR we can consider compatibility upwards to at least Python3.10

Reply:
You cant install 3.8 on all environments, if you require *minimum* 3.8, this can be tested on other environments.

3.8 is by now  quite dated...